src/views/section_title.ts

```diff
@@ -45,10 +49,17 @@ export class SectionTitleView extends BlockView<SectionTitleNode> {
       this.contentDOM.classList.add('empty-node')
       // the first level is hidden
       // other levels are shifted by 1
-      this.contentDOM.setAttribute(
-        'data-placeholder',
-        `${sectionLevel(level)} heading`
-      )
+      if (ParentNode?.node.type === schema.nodes.box_element) {
```

Comment:
if you need to apply that just for the first title, use this `$pos.node($pos.depth - 1).type === schema.nodes.box_element`

Reply:
done